Codeinwp/neve
# Migrate theme mods when switching to a child theme

### Summary
Added the migration function

### Will affect visual aspect of the product
NO

### Screenshots 

### Test instructions
- Activate Neve and change various theme mods in the customizer like font sizes or font family.
- Switch to a neve child theme and see if the theme mods are still the same.



Closes #2578.
